test/runner/lib/manage_ci.py

```diff
@@ -72,42 +72,50 @@ def __init__(self, core_ci):
 
     def wait(self):
         """Wait for instance to respond to ansible ping."""
-        if self.core_ci.platform in ('vyos',):
-            connection_type = 'network_cli'
-        else:
-            connection_type = 'local'
-
-        extra_vars = [
-            'ansible_host=%s' % self.core_ci.connection.hostname,
-            'ansible_port=%s' % self.core_ci.connection.port,
-            'ansible_connection=%s' % connection_type,
-            'ansible_ssh_private_key_file=%s' % self.core_ci.ssh_key.key,
-            'ansible_network_os=%s' % self.core_ci.platform,
-        ]
+        tags = ['all']
+        if self.core_ci.platform in ('eos', 'vyos'):
+            tags.append('cli')
 
         name = '%s-%s' % (self.core_ci.platform, self.core_ci.version.replace('.', '-'))
 
         env = ansible_environment(self.core_ci.args)
-        cmd = [
-            'ansible',
-            '-m', '%s_command' % self.core_ci.platform,
-            '-a', 'commands=?',
-            '-u', self.core_ci.connection.username,
-            '-i', '%s,' % name,
-            '-e', ' '.join(extra_vars),
-            name,
-        ]
-
-        for _ in range(1, 90):
-            try:
-                intercept_command(self.core_ci.args, cmd, 'ping', env=env)
-                return
-            except SubprocessError:
-                sleep(10)
-                continue
-
-        raise ApplicationError('Timeout waiting for %s/%s instance %s.' %
-                               (self.core_ci.platform, self.core_ci.version, self.core_ci.instance_id))
+        for tag in tags:
```

Comment:
Checking host connectivity with one connection type should be sufficient. This logic is only intended to verify the host is ready to accept tests, not to actually test any of the code.

Also, why does the connection type even need to be specified here? Is it now a required value?

Reply:
Right, I understand what that does now... just swapping `local` for `network_cli` as appropriate now